src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -854,14 +854,10 @@ public IEnumerable<Element> GetChildElements()
         private IEnumerable<Element> GetElementChildren(Autodesk.Revit.DB.Element element)
         {
             List<Element> components = new List<Element>();
-            int categoryId = element.Category.Id.IntegerValue;
-            if (!Enum.IsDefined(typeof(BuiltInCategory), categoryId))
+            BuiltInCategory builtInCategory = element.Category.BuiltInCategory;
+            if (builtInCategory == BuiltInCategory.INVALID)
```

Comment:
@wangyangshi @mjkkirschner In 2024 (possibly even 2023...) the BuiltInCategory property was added to facilitate things like this...